packages/xo-server-perf-alert/src/index.js

```diff
@@ -157,6 +157,12 @@ export const configurationSchema = {
       items: {
         type: 'object',
         properties: {
+          all: {
+            title: 'All hosts',
+            type: 'boolean',
+            description: "All hosts will be considered for the alert when 'All hosts' option is enabled.",
```

Comment:
IMHO, the description is next to the toggle, so, no need to add more details.

```suggestion
            description: "When enabled, all hosts will be considered for the alert.",
```
Or
```suggestion
            description: "When enabled, all hosts will be taken into account.",
```